GitReleaseManager.yaml

```diff
@@ -13,6 +13,7 @@ issue-labels-exclude:
  - "r: Obsolete"
  - "r: Rejected"
  - "r: Replaced"
+ - "r: Reverted"
```

Comment:
These allow to keep the corresponding issues or PR in the milestone without listing them in release notes. But it does not fully work: the git release manager tool does not support them to still bear an including tag such as `t: bug` or other `t:` tags. So I had to choose between removing from the milestone the issues which were replaced or reverted, or removing from them their type tag. I have done the later.